web/template/pa-hub/pa-deputy-header.gotmpl

```diff
@@ -1,6 +1,6 @@
 {{ define "pa-deputy-header" }}
 
-    <a href="javascript: history.go(-1)" class="govuk-back-link">Back</a>
+    <a id="back-button" class="govuk-back-link">Back</a>
```

Comment:
Is it worth adding a prefix to the id so we know its not for styling but for functionality? maybe f-back-button